app/models/event_email.rb

```diff
@@ -0,0 +1,8 @@
+class EventEmail < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :event
+
+  validates :to, presence: true
+  validates :subject, presence: true, length: { maximum: 150 }
+  validates :body, presence: true
```

Comment:
We are using [shoulda matchers](http://www.rubydoc.info/github/thoughtbot/shoulda-matchers/Shoulda/Matchers) which provides some nice methods for testing validations.
Please add them to the event mail tests.